src/facade/dragonfly_connection.h

```diff
@@ -294,8 +294,16 @@ class Connection : public util::Connection {
 
   void SetClientTrackingSwitch(bool is_on);
 
+  void LastCommandIsClientCaching();
```

Comment:
please improve the naming. This function sets a variable and yet it has "Is" in its name which implies the opposite.

Reply:
I agree `is` looks like a question than a statement

Reply:
I'd suggest to hide all of this from the general connection interface into a dedicated Tracking().Foo()